debezium-server-redis/src/main/java/io/debezium/server/redis/RedisMemoryThreshold.java

```diff
@@ -104,13 +112,16 @@ private Tuple2<Long, Long> memoryTuple(long defaultMaxMemory) {
         }
 
         Long usedMemory = parseLong(INFO_MEMORY_SECTION_USEDMEMORY, infoMemory.get(INFO_MEMORY_SECTION_USEDMEMORY));
-        Long maxMemory = parseLong(INFO_MEMORY_SECTION_MAXMEMORY, infoMemory.get(INFO_MEMORY_SECTION_MAXMEMORY));
+        Long maxMem = parseLong(INFO_MEMORY_SECTION_MAXMEMORY, infoMemory.get(INFO_MEMORY_SECTION_MAXMEMORY));
```

Comment:
This is definitely not good practice. Please use descriptive names for `maxMem` and `maxMemory` as now itis veru confusing and can be a problem in future maintenance.